modules/core/src/main/scala/org/legogroup/woof/LogInfo.scala

```diff
@@ -2,6 +2,6 @@ package org.legogroup.woof
 
 import java.io.File
 
-case class LogInfo(enclosingClass: String, file: File, lineNumber: Int):
+case class LogInfo(enclosingClass: String, fileName: String, lineNumber: Int):
```

Comment:
JavaScript doesn't support files (what is a file for a browser?) so neither does Scala.js.

If you _know_ your code will not be running in a browser and only on Node.js, which does have files, you can use the `fs2.io.file` package in fs2-io.
https://github.com/typelevel/fs2/blob/main/io/shared/src/main/scala/fs2/io/file/Files.scala